Assets/Photon/PhotonChat/Code/MyClient.cs

```diff
@@ -0,0 +1,148 @@
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Photon.Chat;     
+
+public class MyClient : MonoBehaviour, IChatClientListener 
+{
+    private ChatClient client;       
+    private string AppID = "d0a5737b-396c-4990-8cde-19b4eadbd95e";            
```

Comment:
Where does this come from? Should it be a private thing or are we good to put it on github?